inotify.go

```diff
@@ -88,31 +88,46 @@ func (w *Watcher) Close() error {
 
 // Add starts watching the named file or directory (non-recursively).
 func (w *Watcher) Add(name string) error {
-	name = filepath.Clean(name)
+	return w.AddFlags(name, Create|Remove|Write|Rename|Chmod)
```

Comment:
Would it make sense to add a constant that means "everything" and reference that?  The bitwise or operation of all of these is duplicated in a bunch of places.